aiohttp/test_utils.py

```diff
@@ -433,7 +433,11 @@ def setUp(self) -> None:
         try:
             self.loop = asyncio.get_running_loop()
         except RuntimeError:
-            self.loop = asyncio.get_event_loop_policy().get_event_loop()
+            try:
+                self.loop = asyncio.get_event_loop()
```

Comment:
yeah this is not correct - you'll be getting a new and different loop https://github.com/python/cpython/issues/95736